snuba/datasets/session_events_raw.py

```diff
@@ -0,0 +1,87 @@
+from datetime import timedelta
+from typing import Mapping, Sequence
+
+from snuba.datasets.dataset import TimeSeriesDataset
+from snuba.clickhouse.columns import (
+    ColumnSet,
+    DateTime,
+    LowCardinality,
+    Nullable,
+    String,
+    UInt,
+    Float,
+    UUID,
+)
+from snuba.datasets.schemas.tables import MergeTreeSchema
+from snuba.datasets.dataset_schemas import DatasetSchemas
+from snuba.query.extensions import QueryExtension
+from snuba.query.organization_extension import OrganizationExtension
+from snuba.query.processors.basic_functions import BasicFunctionsProcessor
+from snuba.query.processors.prewhere import PrewhereProcessor
+from snuba.query.query_processor import QueryProcessor
+from snuba.query.timeseries import TimeSeriesExtension
+
+
+class SessionEventsRawDataset(TimeSeriesDataset):
+    def __init__(self) -> None:
+        read_columns = ColumnSet(
+            [
+                ("event_id", UUID()),
+                ("session_id", UUID()),
+                ("distinct_id", UUID()),
+                ("seq", UInt(64)),
+                ("org_id", UInt(64)),
+                ("project_id", UInt(64)),
+                ("retention_days", UInt(16)),
+                ("deleted", UInt(8)),
+                ("sample_rate", Float(32)),
+                ("status", UInt(8)),
+                ("timestamp", DateTime()),
+                ("started", DateTime()),
+                ("user_email", Nullable(String())),
+                ("release", LowCardinality(Nullable(String()))),
+                ("environment", LowCardinality(Nullable(String()))),
+                ("os", Nullable(String())),
+                ("os_version", Nullable(String())),
+                ("device_family", Nullable(String())),
```

Comment:
Why are these nullable ?
Also I suspect that os, os_version and device_family should be lowcardinality as well.